app/build.gradle.kts

```diff
@@ -141,9 +141,12 @@ dependencies {
         libs.pages,
         libs.slf4j.api,
         libs.theme,
-        project(":authentication")
+        libs.authentication,
+        project(":features")
```

Comment:
You may want to play around with type-safe accessors so that as the project grows, it's less to remember as you can lean on auto-complete: https://docs.gradle.org/current/userguide/declaring_dependencies.html#sec:type-safe-project-accessors

Reply:
yep, used type safe `projects.features`